src/Microsoft.DotNet.XHarness.CLI/CommandArguments/WASM/WasmTestCommandArguments.cs

```diff
@@ -10,6 +10,28 @@ namespace Microsoft.DotNet.XHarness.CLI.CommandArguments.Wasm
 {
     internal class WasmTestCommandArguments : TestCommandArguments
```

Comment:
Please note that as it is now the [`TestCommandArguments`](https://github.com/dotnet/xharness/blob/master/src/Microsoft.DotNet.XHarness.CLI/CommandArguments/TestCommandArguments.cs) which the WASM command inherits from already contains some required arguments (such as `--output-directory` that is used by Helix to say where to put results). So these need to be passed as well.

If you don't want that, you need to change the parent class.

Reply:
yeah, thanks, it makes sense to add something first, and then we could get rid of all the redundant things